yarn.lock

```diff
@@ -15133,6 +15271,11 @@ react-is@^17.0.1:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-17.0.2.tgz#e691d4a8e9c789365655539ab372762b0efb54f0"
   integrity sha512-w2GsyukL62IJnlaff/nRegPQR94C/XXamvMWmSHRJ4y7Ts/4ocGRmTHvOs8PSE6pB3dWOrD/nueuU5sduBsQ4w==
 
+react-is@^18.0.0:
```

Comment:
Can we avoid this package, or is it intentional?

Reply:
Seems like a peer-dependency from jest, we already have a couple of entries for this

https://github.com/chatwoot/chatwoot/blob/533f9f7fe4999cd460a59711003c4a42122f2b54/yarn.lock#L15126-L15134

It's a dev dependency, should not show up in our builds